packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -36,8 +36,9 @@ describe('ReactIncrementalErrorHandling', () => {
   }
 
   function stripStackTrace(obj) {
-    obj[0].prop = removeStack(obj[0].prop);
-    return obj;
+    let stripped = [...obj];
+    stripped[0].prop = removeStack(stripped[0].prop);
```

Comment:
This still mutates the object.

Reply:
Okay I guess I don't fully understand mutation. Let me know how the most recent changes look. I'm also going to start trying to fix the build errors.